src/main/java/ch/njol/skript/structures/StructAutoReload.java

```diff
@@ -0,0 +1,233 @@
+package ch.njol.skript.structures;
+
+import ch.njol.skript.ScriptLoader;
+import ch.njol.skript.Skript;
+import ch.njol.skript.SkriptConfig;
+import ch.njol.skript.config.EntryNode;
+import ch.njol.skript.config.Node;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Literal;
+import ch.njol.skript.lang.ParseContext;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.localization.ArgsMessage;
+import ch.njol.skript.localization.Language;
+import ch.njol.skript.localization.PluralizingArgsMessage;
+import ch.njol.skript.log.LogEntry;
+import ch.njol.skript.log.RedirectingLogHandler;
+import ch.njol.skript.log.TimingLogHandler;
+import ch.njol.skript.util.Task;
+import ch.njol.skript.util.Utils;
+import ch.njol.util.OpenCloseable;
+import ch.njol.util.StringUtils;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.logging.Level;
+
+import org.bukkit.Bukkit;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.command.CommandSender;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.jetbrains.annotations.Unmodifiable;
+import org.skriptlang.skript.lang.entry.EntryContainer;
+import org.skriptlang.skript.lang.entry.EntryData;
+import org.skriptlang.skript.lang.entry.EntryValidator;
+import org.skriptlang.skript.lang.script.Script;
+import org.skriptlang.skript.lang.script.ScriptData;
+import org.skriptlang.skript.lang.structure.Structure;
+
+import com.google.common.collect.Lists;
+
+@Name("Auto Reload")
+@Description({
+	"Place at the top of a script file to enable and configure automatic reloading of the script.",
+	"When the script is saved, Skript will automatically reload the script.",
+	"The config.sk node 'script loader thread size' must be set to a positive number for this to be enabled.",
+	"",
+	"available optional nodes:",
+	"\trecipients: The players to send reload messages to. Defaults to console.",
+	"\tpermission: The permission required to receive reload messages. 'recipients' will override this node.",
+})
+@Examples({
+	"auto reload",
+	"",
+	"auto reload:",
+	"\trecipients: \"SkriptDev\", \"61699b2e-d327-4a01-9f1e-0ea8c3f06bc6\" and \"Njol\"", // UUID is Dinnerbone's.
+	"\tpermission: \"skript.reloadnotify\"",
+})
+@Since("INSERT VERSION")
+public class StructAutoReload extends Structure {
+
+	public static final Priority PRIORITY = new Priority(10);
+
+	static {
+		EntryValidator entryValidator = EntryValidator.builder()
+			// Uses OfflinePlayer because this is determined at parse time. Runtime will make sure it's a Player.
+			.addEntryData(new EntryData<OfflinePlayer[]>("recipients", null, true) {
```

Comment:
No matter what I put here, it seems to fail
```
> sk reload test
[12:23:07 INFO]: [Skript] Reloading test.sk...
[12:23:07 INFO]: Line 3: (test.sk)
[12:23:07 INFO]:     Unexpected entry 'recipients: "ShaneBee"'. Check whether it's spelled correctly or remove it
[12:23:07 INFO]:     Line: recipients: "ShaneBee"
[12:23:07 INFO]:  
[12:23:07 INFO]:     Encountered 1 error while reloading test.sk! (53ms)
```

Reply:
Due to canCreateWith being improperly implemented. Instead of creating an entry data from scratch, one of the default implementations can be used. I believe there is one for literals

Reply:
It doesn't properly support arrays